buildres/linux/jabref.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
I would prefer to use the command line options as much as possible. They also apply to the rpm installer.

Reply:
the jpackage options? There is a tradeoff, as i mentioned in the comment
(The launcher should be exactly the same for the rpm, or am I missing something?)

Reply:
Yes, the jpackage options. If the WM option is really necessary, we can keep the custom desktop file in the form of a modified https://hg.openjdk.java.net/jdk/sandbox/file/47f0d21c7e8d/src/jdk.jpackage/linux/classes/jdk/jpackage/internal/resources/template.desktop (I'm guessing the file name here) but should still use the command line options. Moreover, it should be reported to the java devs that such an option would be worthwhile to add (remember jpackage is beta).